package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "binaris",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
just noticed this now, why did you bump the version? there are no code changes in this PR.

Reply:
You're right I'll rebase and remove that change.